sdk/samples/iot/provisioning/README.md

```diff
@@ -44,8 +44,28 @@ Further background on the Azure IoT Client library and key concepts are explaine
 * Have Microsoft [VCPKG](https://github.com/microsoft/vcpkg) package manager and [Eclipse Paho MQTT C client](https://www.eclipse.org/paho/) installed.  Use the directions [here](https://github.com/Azure/azure-sdk-for-c#development-environment) to download VCPKG and install Paho MQTT.
 
 ## Getting Started
+### Environment Variables
+Samples use environment variables for a variety of purposes, including filepaths and connection parameters. Please keep in mind, every time a new terminal is opened, the environment variables will have to be reset.  Setting a variable will take the following form:
 
-All samples require either x509 certification or SAS symmetric key authentication to connect to Azure IoT Hub Device Provisioning Service (DPS).  To easily run this sample, we have provided a script to generate a self-signed device certification used for device authentication.  Three different `.pem` files will be produced.  **This script is intended for sample use only and not to be used in production code**.
+Linux:
+```bash
+export ENV_VARIABLE_NAME=VALUE
+```
+
+Windows:
+```cmd
+set "ENV_VARIABLE_NAME=VALUE"
```

Comment:
nit: I don't think this has to be in quotes (unless maybe there are white space characters).
```suggestion
set ENV_VARIABLE_NAME=VALUE
```